core/src/main/java/com/linecorp/armeria/client/limit/ConcurrencyLimitingHttpClient.java

```diff
@@ -69,7 +69,7 @@ private ConcurrencyLimitingHttpClient(Client<HttpRequest, HttpResponse> delegate
     protected Deferred<HttpResponse> defer(ClientRequestContext ctx, HttpRequest req) throws Exception {
         return new Deferred<HttpResponse>() {
             private final CompletableFuture<HttpResponse> responseFuture = new CompletableFuture<>();
-            private final HttpResponse res = HttpResponse.from(responseFuture);
+            private final HttpResponse res = HttpResponse.from(responseFuture, ctx.eventLoop());
```

Comment:
How about extracting `ctx.eventLoop()` into a final local variable of `defer()`, so that the anonymous `Deferred` does not have the reference to the context but only to the event loop?

Reply:
Fixed, thanks!